README.md

```diff
@@ -34,14 +34,16 @@ GUARD Tool
 Usage: guard [OPTIONS]
 
 Options:
-  -h,--help              Use the below listed functions.
-                         Warning: Don't try guard on non guardable units
-                         (sys, perv)
-  -c,--create TEXT       Create GUARD record, expects physical path as input.
-  -d,--delete TEXT       Delete GUARD record, expects physical path as input.
-  -l,--list              Listing of GUARDed resources.
-  -r,--clearall          Clears GUARD states for all resources.
-  -v,--version           Version of GUARD tool.
+  -h,--help                Use the below listed functions.
+                           Warning: Don't try guard on non guardable units
+                           (sys, perv)
+  -c,--create TEXT         Create GUARD record, expects physical path as input.
+  -d,--delete TEXT         Delete GUARD record, expects physical path as input.
+  -l,--list                Listing of GUARDed resources.
+  -r,--clearall            Clears GUARD states for all resources.
+  -s,-clearresolvedrecord  Delete GUARD states for oldest resolved resource
```

Comment:
Yes, as per discussion made this change and clearing the oldest guard record, this is basically to not lose the history of resolved records.

Reply:
That is production use case , am refering guard tool usecase.

Reply:
This option is removed because of the changes in clearResolvedRecord function i.e.
Hostboot removes the resolved/invalid guard record and writes the new record in the same location. Hence providing this as an option in tool can create problem. Hence removing it.

Reply:
This option did not implement in this patch. - You forget remove?